src/applications/search/constants/index.js

```diff
@@ -0,0 +1,5 @@
+export const BASE_URI = 'https://search.usa.gov/api/v2/search';
+export const AFFILIATE = 'vets.gov_search';
```

Comment:
I think this should be 'va' - - right @hpjaj ?

Reply:
@kfrz Are these deprecated because we are using vets-api for the endpoint now instead of hitting search.gov directly? I'm not using any of the constants here except for page size